pkg/configmap/services/services.go

```diff
@@ -85,6 +87,29 @@ func NewServicesConfigMapData(services *[]Service, files *[]FileInfo) (*services
 	return cmData, nil
 }
 
+// GenerateWindowsServicesConfigMap creates an immutable service ConfigMap which provides WICD with the specifications
+// for each Windows service that must be created on a Windows instance.
+func GenerateWindowsServicesConfigMap(name, namespace string) *core.ConfigMap {
```

Comment:
This commit seems like it belongs in the other PR, where this would actually be used

Reply:
True. I kept going back and forth whether include it or not haha